libs/utils/utils.js

```diff
@@ -754,6 +754,13 @@ export function decorateImageLinks(el) {
   });
 }
 
+export const setDialogAndIframeTitle = ({ element, title }) => {
+  const dialogModal = element.closest('.dialog-modal');
+
+  if (element && title) element.title = title;
+  if (dialogModal && title) dialogModal.setAttribute('aria-label', title);
```

Comment:
How about this to keep it short:
```suggestion
  if (!element || !title) return;
  element.title = title;
  element.closest('.dialog-modal')?.setAttribute('aria-label', title);
```